js/game.js

```diff
@@ -0,0 +1,58 @@
+
+class Game {
+    constructor (ctx, width, height, player) {
+  this.ctx = ctx;
+  this.width = width;
+  this.height = height;
+  this.player = player;
+  this.intervalId = null;
+  this.frames = 0;
+
+  }
+    start() {
+  
+  this.intervalId = setInterval(this.update, 1000 / 60)
+    }
+  
+    update = () => {
+  
+   this.frames++;
+   this.clear();
+   this.player.newPos();
+   this.player.draw();
+   this.updateEnemies();
+   this.checkGameOver();
+    }
+  
+    stop () {
+      clearInterval(this.intervalId);
+    }
+  
+    clear () {
```

Comment:
One way that we have to draw to road is to do it right here. Instead of clearing the canvas with a blank rectangle we can do it with the road image, and in this way we will be taking care of 2 things with just 1 method:

```
  clear() {
    let road = new Image();
    road.src = "images/road.png";
    this.ctx.drawImage(road, 0, 0, this.width, this.height);
  }
```